mealie/run.sh

```diff
@@ -7,20 +7,18 @@ ARG1=${1:-production}
 # DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )" 
 
 # # Initialize Database Prerun
-poetry run python /app/mealie/db/init_db.py
+poetry run alembic upgrade head
 poetry run python /app/mealie/services/image/minify.py
 
-# Migrations
 # TODO
-    # Migrations
-    # Set Port from ENV Variable
+# Set Port from ENV Variable
 
-if [[ "$ARG1" = "reload" ]]
```

Comment:
If the application is constantly reloading you need to properly setup you development environment by setting the ENV variables, not remove the code for development. 

```env
# Determines Production Mode, This will set the directory path to use for data storage
PRODUCTION=False
```

Reply:
Not sure how this is supposed to work, I'll put it back.